src/robusta/integrations/prometheus/trigger.py

```diff
@@ -11,10 +11,27 @@
 from robusta.integrations.kubernetes.custom_models import RobustaDeployment, RobustaJob, RobustaPod
 from robusta.integrations.prometheus.models import PrometheusAlert, PrometheusKubernetesAlert
 from robusta.utils.cluster_provider_discovery import cluster_provider
+from robusta.utils.scope import ScopeParams, BaseScopeMatcher
+
 
 ALERT_EVENT = "alert_event"
 
 
+class PrometheusTriggerEventScopeMatcher(BaseScopeMatcher):
+    def __init__(self, data):
+        # We put labels data both on the top level and under the "labels" key,
+        # so that keys like alertname, severity etc. are accessible without
+        # labels: ... matching.
+        self.data = data.labels.copy()
+        self.data["annotations"] = data.annotations
+        self.data["labels"] = data.labels.copy()
+        # status is the only field of PrometheusAlert that we want to expose.
+        self.data["status"] = data.status
```

Comment:
Not sure we want the `status` here, because we have the `status` filter on the trigger

Reply:
refactored as discussed